app/controllers/admin/questions_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Admin::QuestionsController < AdminController
+  def index
+    @bank_questions = Question.for_banks
+    @partner_questions = Question.for_partners
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.create(question_params)
+    if @question.valid?
+      redirect_to admin_questions_path
+    else
+      flash[:error] = "Failed to create question. #{error_message_for_answer(@question.errors)}"
+      render :new
+    end
+  end
+
+  def edit
+    @question = current_question
+  end
+
+  def update
+    @question = current_question
+    @question.update(question_params)
+    if @question.valid?
+      redirect_to admin_questions_path
+    else
+      flash[:error] = "Failed to update question. #{error_message_for_answer(@question.errors)}"
+      render :edit
+    end
+  end
+
+  def destroy
+    @question = current_question
+    @question.destroy
+    redirect_to admin_questions_path
```

Comment:
Notably, a `destroy` can fail for various reasons. Could you add the `if/else` to ensure that we notify the user if the destroy action failed?

Reply:
Rubocop didn't like when I tried to use if/else so I came up with something a little shorter that should achieve what you mentioned here.